net-im/qtox/metadata.xml

```diff
@@ -10,7 +10,7 @@
 		<name>Proxy Maintainers</name>
 	</maintainer>
 	<use>
-		<flag name="gtk">Adds support for GTK and Status Notifier tray backends</flag>
+		<flag name="notifications">Adds support for GTK Status Notifier tray backends</flag>
 		<flag name="X">Adds support for X11</flag>
```

Comment:
What does this do, exactly? Apparently Qt itself provides most of X11 support, so I suppose this provides some extra features.

Reply:
You didn't answer this.

Reply:
It uses GTK to provide desktop notifications (as said in the flag description). That's why the flag was called `gtk` before.

Reply:
I was asking about USE=X.

Reply:
It is not used by the ebuild currently, but it should include `-DPLATFORM_EXTENSIONS=$(usex X)`.
The flag enables X11 extensions (xscreensaver, which is seems to be used to retrieve the user's idle time, and switch to "away" status if needed).